lib/memory-manager.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+
+// const LRU = require('lru-cache');
+const md5 = require('md5');
+const logger = require('@janiscommerce/logger');
+const LRU = require('lru-cache');
```

Comment:
```suggestion
const LRU = require(path.join(process.cwd(), 'node_modules', 'lru-cache'));
```
Es necesario agregar las rutas a las dependencias dentro del process cwd ya que no estan como dependencias del paquete.